qiita_ware/test/test_commands.py

```diff
@@ -145,14 +146,13 @@ def test_submit_EBI_parse_EBI_reply_failure(self):
             submit_EBI(ppd.id, 'VALIDATE', True)
 
     def test_full_submission(self):
-        artifact = self.generate_new_study_with_preprocessed_data()
-
-        # just making sure
-        self.assertEqual(artifact.study.ebi_submission_status, 'not submitted')
-
-        submit_EBI(artifact.id, 'VALIDATE', True, test=True)
-
-        self.assertEqual(artifact.study.ebi_submission_status, 'submitted')
+        ascp_pass = environ.get('ASPERA_SCP_PASS', '')
```

Comment:
I think it would make sense to decorate this test with `@unitttest.skipIf` and check for the existence of the password:

https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures

Reply:
First time I hear of this, pretty cool, adding.